### PR TITLE
Updating TEFCA readme.md

### LINKED ISSUE #1937: Update TEFCA Viewer Documentation
- Still says eCR viewer in a lot of places 
- Writing out directions for how to work with locally using `npm run dev` etc. could be helpful

### PR DESCRIPTION
**Pull request by robertandremitchell**
# PULL REQUEST

## Summary
Updates some references from ECR to TEFCA. Also trying to debug some docker steps.

I can build the docker image, run `docker run -p 8080:8080 tefca-viewer` but neither the `localhost:3000` or `localhost:8080` currently resolve.

The updates made make it so `docker run -p 3000:3000 tefca-viewer` does work. Another thing I noticed is that, currently, `localhost:3000` goes to a Page Not Found. the purpose of `middleware.js` is to redirect it to `/tefca-viewer`.

This...kinda works, at the moment. it does a double append that I for the life of me can't figure out. However, it does work, and does not seem to create a new issue. I'll keep trying to debug but if there's someone with more familiarity with this kind of redirect work, I would be happy to pair.

I think the main issue is that the main page is not coded (the default page is `/tefca-viewer` based off the `basePath` set in `next.config.js`), so I think we just need to...somewhere in the code make that the main page or alias the main page to be /tefca-viewer. I'm just not sure on best practices, but that would then allow us to remove the `middleware.js`. If we comment out `basePath`, the default page works, but then we have to change the header redirect. But that does seem to work.

Not sure what we would want to do there, but I feel I am starting to get far afield, so curious what people think of this update to the README and the initialization/updating of docker/docker compose, to start.

## Related Issue
Fixes #1937 

## Additional Information
Do we want Docker to be the recommended versus the Node.js source code? Both currently have language calling them the recommended method.
Do we want to add a simplified docker compose as part of this work? Right now the docs have it in there, but I don't know if we need it currently. I've added a dummy one.
Will we actually need a seed database? How is our test data currently populating in TEFCA?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/tefca-viewer/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "test": "npm run test:unit && npm run test:integration",
     "test:unit": "TZ=America/New_York jest",
     "test:unit:watch": "jest --watch",
-    "test:integration": "mocha --timeout 600000 tests/integration/test_ecr-viewer.js",
+    "test:integration": "mocha --timeout 600000 tests/integration/test_tefca-viewer.js",
     "convert-seed-data": "sh ./seed-scripts/convert.sh",
```
[the remark this answers is not in the document]
Comment (robertandremitchell) [reply containers/tefca-viewer/package.json]: updated!

File: containers/tefca-viewer/README.md [hunk 2]
```diff
@@ -19,39 +19,33 @@ To run the eCR Viewer app with Docker, follow these steps.
...
 1. Ensure that both Git and Node 18.x or higher are installed.
 2. Clone the PHDI repository with `git clone https://github.com/CDCgov/phdi`.
-3. Navigate to `/phdi/containers/ecr-viewer/`.
-4. Install all of the Node dependencies for the eCR Viewer app with `npm install`.
-5. Run the eCR Viewer app on `localhost:3000` with `npm run dev`.
+3. Navigate to `/phdi/containers/tefca-viewer/`.
+4. Install all of the Node dependencies for the TEFCA Viewer app with `npm install`.
+5. Run the TEFCA Viewer app on `localhost:3000` with `npm run dev`.
 
 ### Building the Docker Image
```
[the remark this answers is not in the document]
Comment (robertandremitchell) [reply containers/tefca-viewer/README.md:37]: I am not sure what we want to include as the recommended service. For now, it feels like `npm run dev` is simplest, but if we start to incorporate other building blocks, I would think `docker compose` would be.

curious for thoughts from other reviewers.

[the remark this answers is not in the document]
Comment (DanPaseltiner) [reply containers/tefca-viewer/README.md:37]: Currently not _needed_, but we deploy currently by shipping a docker image to K8s in phdi-playground. A couple times when I have needed to debug I found it helpful to build and run the image locally to be sure I was looking at the exact same version as was deployed.

[the remark this answers is not in the document]
Comment (DanPaseltiner) [reply containers/tefca-viewer/README.md:37]: Also, I think pretty soon we will likely need to add in a database and integration with the TCR. At that point docker compose will definitely be our friend.

File: containers/tefca-viewer/package.json
Comment (DanPaseltiner) [inline containers/tefca-viewer/package.json:1]: nice

File: containers/tefca-viewer/README.md [hunk 3]
```diff
@@ -19,39 +19,33 @@ To run the eCR Viewer app with Docker, follow these steps.
 Docker version 20.10.21, build baeda1f
 ```
 
-2. Download a copy of the Docker image from the PHDI repository by running `docker pull ghcr.io/cdcgov/phdi/ecr-viewer:latest`.
-3. Run the service with `docker run -p 8080:8080 ecr-viewer:latest`.
+2. Download a copy of the Docker image from the PHDI repository by running `docker pull ghcr.io/cdcgov/phdi/tefca-viewer:latest`.
+3. Run the service with `docker run -p 3000:3000 tefca-viewer:latest`.
```
[the remark this answers is not in the document]
Comment (DanPaseltiner) [reply containers/tefca-viewer/README.md]: This isn't a windows machine thing this is just how Docker works. The docs were wrong before.

**Review by DanPaseltiner — APPROVED** [review]

**Pull request merged**